firebase-common/firebase-common.gradle

```
@@ -61,6 +61,7 @@ android {
 }
 
 dependencies {
+    implementation project(':firebase-components')
```

I think with this:
```suggestion
    api project(':firebase-components')
```
you don't have to add the dependency to all the other SDKs that already depend on this one.

---

Nice, I did not know you could suggest changes in github!

As for changing to `api`, I explicitly decided not to do it and force SDKs to include a dependency on components so that it propagates into their pom.xml as opposed to transitively depending on it.

---

OK, fair enough.